src/utils/makeReactNativeConfig.js

```diff
@@ -187,6 +187,10 @@ const getDefaultConfig = ({
               test: /\.(js|css|(js)?bundle)($|\?)/i,
               filename: '[file].map',
             }),
+            new webpack.BannerPlugin({
+              banner: 'if (this && !this.self) { this.self = this; };\n',
+              raw: true,
+            }),
```

Comment:
looks like this could be moved to common webpack config (along with the one on line 175)

Reply:
Thanks for reminding.